app/gui/suggestion-database/src/entry.rs

```diff
@@ -888,7 +888,7 @@ fn resolve_tag_value<'a>(
     db: &SuggestionDatabase,
     parser: &parser::Parser,
 ) -> TagValueResolution<'a> {
-    let stripped_expr = raw_expression.trim_start_matches('(').trim_end_matches(')');
+    let stripped_expr = raw_expression.trim_matches('(').trim_matches(' ');
```

Comment:
```suggestion
    let stripped_expr = raw_expression.trim_matches(['(', ')', ' ']);
```
We lost closing `)` somewhere. And I think spaces outside can also be trimmed.